CONTRIBUTING.md

```diff
@@ -54,6 +57,27 @@ approach is to keep the *one step = one commit* approach.
 Avoid merge commits in pull requests - use `git rebase` to get rid of them. This helps us keep git
 tree clean and helps to make it easy to understand historical changes done in past commits.
 
+
+### Maintainer Information
+
+Reviewers should pay attention to the following points:
+
+The PR ...
+* ... has been approved by an SME of the area that the PR addresses, which indicates that it is generally a step in a right direction.
+* ... passes PR gating tests, or failing tests are waived in a comment by the reviewer explaining the reasoning.
+* ... adheres to the [coding style](docs/manual/developer/04_style_guide.md#complianceascode-style-guide).
+* ... has been tested. The following options that allow for testing various aspects of the project:
+  * [SSG Test Suite](tests/README.md) for rule tests to ensure that OVAL checks are correct and test the ability of remediations to satisfy those checks.
+  * [BATS framework](tests/unit/bash) for bash tests that allows for fast and exhaustive testing of remediations that are parametrized by Jinja2.
+  * [Unit tests](tests/unit) that test components of the build system as well as components of the SSG Test Suite.
+  * Ad-hoc tests that are integrated into the `ctest` chain directly, i.e. the shellcheck test.
+* ... updates READMEs, man pages or other documentation when changes of described behavior are introduced.
+* ... doesn't contain merge commits - those can be removed by rebasing.
```

Comment:
Might be helpful to link to a rebasing/git guide if we have one.

Reply:
We don't, please somebody suggest something that is both to the point and isn't located on some obscure blog that may go offline in a couple of years.

Reply:
@matejak , this might be useful: https://github.com/marcusburghardt/ansible-role-openscap/blob/master/files/STARTGUIDE.md

It is included in the [ansible-role-openscap](https://galaxy.ansible.com/marcusburghardt/ansible_role_openscap).

Reply:
So for the time being, I would proceed with merging this PR as it is, because your guide doesn't contain all the info that one could be after, and supplying that would take some time. I see no issue with referencing it, or even with putting it here and referencing it from the roles project.

I think that following issues should be covered there:
- merge commits, how to recognize them
- what rebase does, and what can be accomplished by interactive rebase (reordering, merge of commits)
- a few words about conflict resolution - when to expect them and generally how to handle them

Reply:
We can also turn on a setting in the project that forbids merge commits to help automate this requirement.